dotcom-rendering/src/components/Titlepiece.tsx

```diff
@@ -0,0 +1,340 @@
+	{ name: 'Obituaries', slug: 'obituaries' },
+] as const satisfies ReadonlyArray<{
+	name: string;
+	slug: string;
+}>;
+
+const getWindowWidth = () => window.innerWidth;
+
+const veggieBurgerDiameter = 40;
+
+const pillarLinkWidth740 = 108;
+const pillarLinkWidth1280 = 125;
+const pillarLinkWidth1440 = 136;
+
+const editionsMenuStyles = css`
+	grid-column: content-start / content-end;
+	grid-row: 1;
+	@media (min-width: 375px) {
+		justify-self: end;
+	}
+`;
+
+const guardianLogoStyles = css`
+	grid-column: content-start / content-end;
+	grid-row: 1;
+	justify-self: end;
+	align-self: end;
+	margin-top: ${space[2]}px;
+	margin-bottom: 6px;
+	margin-right: ${veggieBurgerDiameter + space[3]}px;
+	@media (min-width: 375px) {
+		margin-right: 0;
+	}
+	svg {
+		height: 49px;
+		@media (min-width: 375px) and (max-width: 739px) {
+			height: 67px;
+		}
+		@media (min-width: 740px) and (max-width: 1023px) {
+			height: 96px;
+		}
+		@media (min-width: 1024px) and (max-width: 1279px) {
+			height: 94px;
+		}
+		@media (min-width: 1280px) and (max-width: 1439px) {
+			height: 115px;
+		}
+		@media (min-width: 1440px) {
+			height: 135px;
+		}
+	}
+	@media (min-width: 480px) and (max-width: 1023px) {
+		margin-bottom: 10px;
+	}
+	@media (min-width: 1024px) and (max-width: 1279px) {
+		margin-bottom: ${space[3]};
+	}
+	@media (min-width: 1280px) {
+		margin-bottom: 14px;
+	}
+`;
+
+const burgerStyles = css`
+	grid-column: content-start / content-end;
+	grid-row: 1;
+	justify-self: end;
+	align-self: end;
+	height: ${veggieBurgerDiameter}px;
+	width: ${veggieBurgerDiameter}px;
+	margin-bottom: 6px;
+	border-radius: 50%;
+	background-color: ${themePalette('--masthead-veggie-burger-background')};
+	justify-content: center;
+	display: flex;
+	:hover {
+		background-color: ${themePalette(
+			'--masthead-veggie-burger-background-hover',
+		)};
+	}
+	@media (min-width: 375px) {
+		grid-row: 2;
+		align-self: center;
+	}
+	@media (min-width: 1024px) {
+		grid-row: 1 / 2;
+		align-self: end;
+		justify-self: start;
+		margin-left: ${pillarLinkWidth740 * pillars.length + space[1]}px;
+		margin-bottom: ${space[3]}px;
+	}
+	@media (min-width: 1280px) {
+		margin-left: ${pillarLinkWidth1280 * pillars.length + space[1]}px;
+	}
+	@media (min-width: 1440px) {
+		margin-left: ${pillarLinkWidth1440 * pillars.length + space[1]}px;
+	}
+`;
+
+const navLinkStyles = css`
+	a {
+		color: ${themePalette('--masthead-nav-link-text')};
+		text-decoration: none;
+	}
+`;
+
+const pillarsNavStyles = css`
+	grid-column: content-start / viewport-end;
+	grid-row: 2;
+	align-self: end;
+
+	${headlineBold14}
+	margin-top: ${space[2]}px;
+	border-bottom: 1px solid ${themePalette('--masthead-nav-lines')};
+	@media (min-width: 1024px) {
+		grid-row: 1 / 2;
+	}
+	ul {
+		display: flex;
+	}
+	li {
+		height: 28px;
+		padding-right: ${space[1]}px;
+		@media (min-width: 375px) and (max-width: 479px) {
+			height: 34px;
+		}
+		@media (min-width: 480px) and (max-width: 739px) {
```

Comment:
Ah good catch, thank you @mxdvl. Is this the kid of implementation you had in mind: https://github.com/guardian/dotcom-rendering/pull/11368/commits/76b4ce6cad762ea528a8be68b589e5e83cfb69a3?

A lot of the design break points are at widths not covered in Source (or are a mix, e.g. a Source-friendly 'from' but a bespoke 'until'). Is there a way to mix the two so we're getting as much standardisation as possible?

Reply:
Ok, we've clarified that the breakpoints in Figma weren't prescriptive, more 'this is what it would look like at this size'. The design pixel headings have been updated to align with Source and all breakpoints in the PR now use it